dso-l2/src/main/java/com/tc/objectserver/persistence/Persistor.java

```diff
@@ -64,8 +65,8 @@ public Persistor(IPersistentStorage persistentStorage) {
   public void start() {
     sequenceManager = new SequenceManager(persistentStorage);
     clientStatePersistor = new ClientStatePersistor(sequenceManager, persistentStorage);
-
-    if (!this.clusterStatePersistor.isDBClean()) {
+    wasDBClean = this.clusterStatePersistor.isDBClean();
```

Comment:
Do we still want to do this, here, or can we rely on the server forcing us into a clean state, from the top?